### DISCUSSION TITLE
Request for CIS Azure Kubernetes Service (AKS) Benchmark v1.0.0

### DISCUSSION THREAD

**Discussion by ankitsrao** [aquasecurity/kube-bench#937] [Questions and Help] [answered]
Hi Team,
I just came across the CIS Azure Kubernetes Service (AKS) Benchmark v1.0.0, which was published recently in March 2021. I could not find any document/ feature request for kube-bench support for the same.
I would request if we could plan to add support for this new AKS benchmark in kube-bench?

- TIA

**Comment by yoavrotems** [comment, marked as answer]
We do support aks https://github.com/aquasecurity/kube-bench/pull/733 which we got contributed from @borkod . 
Its a good idea though to compare it and support the CIS instead or both depends on the differences. 
I will open a issue in the subject if someone is up to do it 😃
